### DISCUSSION TITLE
Releases/tags

### DISCUSSION THREAD

**Discussion by Jogai** [leonardoporro/Detached-Mapper#111] [Ideas]
Its a bit hard to see what is changed. Maybe tagging the releases would help?

**Comment by leonardoporro** [comment]
Hi @Jogai, sure
I was supposed to complete this, but didn´t have the time before:

[Changes & Breaking Changes] (https://github.com/leonardoporro/Detached-Mapper/wiki/Changes-&-Breaking-Changes)

I'm going to try to be more organized. Usually work on this late in the night or while doing other things.

**Reply by Jogai** [reply]
This is great. I completely understand.
